### PR TITLE
Removed go1.3.3 support

### LINKED ISSUE #12549: remove go1.3.3 support
this was added for go1.3.3 support but we no longer need, https://github.com/docker/docker/blob/master/pkg/requestdecorator/requestdecorator_test.go#L10

there are other instances places as well. finding them all would be great but some would be awesome.

### PR DESCRIPTION
**Pull request by fntlnz**
fix #12549

Signed-off-by: Lorenzo Fontana fontanalorenzo@me.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
can you change your comment to fix # so it closes the ticket

**Comment by fntlnz** [comment]
@jfrazelle Yes no problem, I thought you wanted to keep it open to investigate further

**Comment by jfrazelle** [comment]
ya no worries we can open a new one if so

File: Dockerfile [hunk 1]
```diff
@@ -98,9 +98,9 @@ RUN cd /usr/local/go/src \
 			./make.bash --no-clean 2>&1; \
 	done
 
-# We still support compiling with older Go, so need to grab older "gofmt"
-ENV GOFMT_VERSION 1.3.3
-RUN curl -sSL https://storage.googleapis.com/golang/go${GOFMT_VERSION}.$(go env GOOS)-$(go env GOARCH).tar.gz | tar -C /go/bin -xz --strip-components=2 go/bin/gofmt
+# This has been commented out and kept as reference because we don't support compiling with older Go anymore.
+# # ENV GOFMT_VERSION 1.3.3
```
Comment (jfrazelle) [inline Dockerfile]: nit, but why 2 `#`

**Comment by jfrazelle** [comment]
LGTM ping @tianon

**Comment by jfrazelle** [comment]
can you squash your commits into 1

**Comment by jfrazelle** [comment]
https://docs.docker.com/project/review-pr/#update-an-existing-pull-request

**Comment by fntlnz** [comment]
@jfrazelle Done :)

**Comment by tianon** [comment]
LGTM

**Pull request merged**